xbmc/cores/AudioEngine/Sinks/pipewire/AESinkPipewire.cpp

```diff
@@ -495,9 +495,6 @@ unsigned int CAESinkPipewire::AddPackets(uint8_t** data, unsigned int frames, un
 
   size_t length = frames * m_format.m_frameSize;
 
-  if (spaData->maxsize < length)
-    length = spaData->maxsize;
```

Comment:
Hehe - if it's just a "should" - keep it in :-) but with the 6 buffers we use in future, while blocking on 4 - all should be fine.

Reply:
we define the buffer size based on the period, so unless AE ever sends data larger than one period we are fine.